ArkhamOverlay/Pages/SelectCards/SelectCardsController.cs

```diff
@@ -109,31 +157,53 @@ public class SelectCardsController : Controller<SelectCardsView, SelectCardsView
         /// <param name="options">The options the user may selectd from when clicking this button</param>
         /// <param name="callback">What to do when the user selects an option</param>
         /// <returns>A list of menu items</returns>
-        private IEnumerable<RightClickOptionCommand> CreateRightClickOptions(IEnumerable<ButtonOption> options, Action<string> callback) {
+        private IEnumerable<RightClickOptionCommand> CreateRightClickOptions(IEnumerable<ButtonOption> options, Action<ButtonOption> callback) {
             var commands = new List<RightClickOptionCommand>();
             foreach (var option in options) {
-                var text = option.GetTextResolvingPlaceholders(ResolveMenuItemPlaceholder);
+                var text = option.GetText(this);
                 if (!string.IsNullOrEmpty(text)) {
-                    commands.Add(new RightClickOptionCommand(option.Option, text, callback));
+                    commands.Add(new RightClickOptionCommand(option, text, callback));
                 }
             }
 
             return commands;
         }
 
         /// <summary>
-        /// Callback to resolve placeholder in a menu item so we can provide more contextual information
+        /// Used by button option to get a name for the card group when displaying an option
```

Comment:
Very minor, but since these are implemented explicitly, you don't really need the `///` comments here. Keeping them is good, but just for future reference

Reply:
got it